Bowling/Bowling/FrameNoBonusRollFactory.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BowlingApp
+{
+    public class FrameNoBonusRollFactory
+    {
+        public const int totalPinsCount = 10;
+        public int PlaceCountHolder { get; set; }
+        
+        public Frame Create(int frameNumber, string bowlingMarks, int placeCount)
```

Comment:
Thinking about it some more, a lot of the complexity of this program is derived from the `placeCount` variable.  Knowing the current frame and the bowling marks should be satisfactory for computing the score.

Reply:
I have removed the dependency on the placeCount variable...and yes it did add too much complexity.  I may have been able to write a lot less code without that in my design.